### ISSUE TITLE
Docker: Plugin installation did not happen in 3.1.1

### ISSUE THREAD

**Issue by Apollon77** [fg2it/grafana-on-raspberry#11]
hey,

I used the following to create the 3.1.1 docker process:

```
docker run -d -p 3000:3000 --name grafana3 -v grafana-storage3:/var/lib/grafana -e "GF_INSTALL_PLUGINS=grafana-clock-panel,grafana-piechart-panel" fg2it/grafana-armhf:v3.1.1
```

The plugins were not installed automatically ...

I was able to use

```
docker exec grafana3 grafana-cli plugins install grafana-clock-panel
docker exec grafana3 grafana-cli plugins install grafana-piechart-panel
```
to install the plugins manually and now they are available ...

Will try now with v4.0.1 -> works there, so only 3.1.1 affected!

**Comment by Apollon77** [comment]
BTW: The original docker repo for docker added " to the plugin install process to allow spaces for defined version installation of plugins:

```
if [ ! -z "${GF_INSTALL_PLUGINS}" ]; then
```

**Comment by fg2it** [comment]
@Apollon77 thanks for the detailed report.
Yes, that was the problem. Had been fixed in [#59](https://github.com/grafana/grafana-docker/commit/bbb2ec58d9b1cb28a9cc9d0f8c396b1f2dad8592).

Since I followed official grafana docker, my images had the same problem as can be seen inspecting [run.sh](https://github.com/fg2it/grafana-on-raspberry/blob/v3.1.1-wheezy-jessie/docker/v3.x/run.sh#L33) script run at start up. I fixed it in [v4.x](https://github.com/fg2it/grafana-on-raspberry/blob/v4.0.1/docker/v4.x/run.sh#L33) so all v4 docker images should be fine.

I chose not to update previous docker images. It would be to costly for my to check and update them.

**Comment by Apollon77** [comment]
Ok, that way ... 3.1.1 was only the "compatibility test" ... currently try 4.0.1 :-)
